docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -191,14 +218,52 @@ Use the following table to populate your datasource.
 
 Note, no username properties are given, because the authentication module only requires the API-KEY.
 
-### SSL Certificate
+### **Configuring a BulkData's Liberty Datasource with API key**
 
-The Db2 certificate should be added to the Liberty Profile truststore. *Be sure to use the same Java runtime that Liberty Profile uses when manipulating any keystores.*
+The IBM FHIR Server Bulk Data modules utilize Java Batch (JSR-352) from the Liberty Profile feature - `batch-1.0`. The batch feature is configured to use Db2 as follows:
 
+1. Create a Db2 user (e.g. FHIRBATCH)
 
-### Encrypt Secrets
+1. and associate it with a ServiceId (no need to create an Administration user, a simple user has sufficient privileges). 
+
+1. Using a valid API-KEY for the given ServiceId, configure a new datasource and the Java Batch persistence layer as follows:
+
+``` xml
+    <dataSource id="fhirbatchDS" jndiName="jdbc/fhirbatchDB">
+        <jdbcDriver libraryRef="fhirSharedLib"/>
+        <properties.db2.jcc
+            serverName="dashdb-txn-flex-************.services.dal.bluemix.net"
+            portNumber="50001"
+            apiKey="<API-KEY>"
+            securityMechanism="15"
+            pluginName="IBMIAMauth"
+            databaseName="BLUDB"
+            currentSchema="JBATCH"
+            driverType="4" sslConnection="true" sslTrustStoreLocation="resources/security/dbTruststore.jks" sslTrustStorePassword="<TRUSTSTORE-PASSWORD>"/>
+    </dataSource>
+
+    <batchPersistence jobStoreRef="BatchDatabaseStore" />
+    <databaseStore id="BatchDatabaseStore" dataSourceRef="fhirbatchDS" schema="JBATCH" tablePrefix="" />
+```
+
+- Note: The Java Batch is configured in `batchDs.xml` and included from the IBM FHIR Server's `server.xml` which is installed to the `{wlp}/usr/server/fhir-server`.
```

Comment:
```suggestion
- Note: The Java Batch is configured in `batchDs.xml` and included from the IBM FHIR Server's `server.xml` which is installed to `{wlp}/usr/server/fhir-server/`.
```

Reply:
I'm grabbing this in my local commit for simplicity